hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HDFSCommands.md

```diff
@@ -722,8 +724,6 @@ Usage: `hdfs debug verifyEC -file <file>`
 |:---- |:---- |
 | [`-file` *EC-file*] | HDFS EC file to be verified. |
 
-Verify the correctness of erasure coding on an erasure coded file.
-
```

Comment:
why is this removed?

Reply:
yeah, deleted by mistake here, it has been fixed.